src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -248,14 +306,18 @@ class EmailNotificationForm extends Component {
                       checked={this.state.agree_to_emails}
                       onChange={() => this.handleCheckboxInput()}/>
                     <label className="agree-to-emails-checkbox label" htmlFor="agree_to_emails">
-                      Sign up for email notifications about upcoming elections in my state.
+                      <FormattedMessage
+                        id="emailNotificationForm.agreeToEmailsMessage"
+                        defaultMessage="Sign up for email notifications about upcoming elections in my state." />
                     </label>
                   </div>
                   <div className="submit-button-container">
                     <button
                       className="submit-button"
                       onClick={(event) => this.handleSubmit(event)}>
-                        Submit
```

Comment:
Just scrolling through here and thinking about what a pain it must have been to find every single place that needed to be translated.  Thank you so much for all of your hard work on the translations @ivonne-hernandez 🥰